### ISSUE TITLE
UX Samples do not scale with the screen well - tested on Iphone 7

### ISSUE THREAD

**Issue by ROBYER1** [Unity-Technologies/arfoundation-demos#33]
Try 800 x 480 in Game view or 1280 x 720 and the bottom anchored overlay is off screen, also the Top of screen content is then simply huge and overlapping the bottom anchored content.

Iphone 7 is 1334 x 750 Pixels.

I am shocked this was released without a simple test of the resolutions in the Game view you can choose by default when making an IOS or Android project in Unity with ARFoundation.

The first device I tested on was an Iphone 7 which experienced these issues unfortunately this solution does not scale well.

Setting the Canvas Scaler to scale with screen size just means that the bottom anchored popups are off the screen always but scaling with it.

Not impressed with the scaling, however the icons are very nice and the presentation is great when it is on the right size phone screen.

**Comment by ROBYER1** [comment]
Also the top and bottom anchors do not play well with Landscape view switching, looks awful.

**Comment by DanMillerDev** [comment]
Thanks for the report, I'll look into adjusting some anchors and screen scaling for smaller devices.

**Comment by DanMillerDev** [comment]
I've made some [changes](https://github.com/Unity-Technologies/arfoundation-demos/commit/9bedd2b32b5bb09878344451d52e4498852d87ef) to the ScreenspaceUI prefab specific to scaling and some small adjustments to positioning. I tested with the device simulator for iPhone 7. See attached Screenshot

![DeviceSimScalingFix](https://user-images.githubusercontent.com/2120584/94504726-4d772500-01be-11eb-850c-d272de74f39d.JPG)


Both the top anchor videos and the reasons should fit properly on all devices now.

I did some testing with landscape mode but was not able to come up with a solution that fit both the videos and reasons properly. The original layout and design was made for portrait mode applications so for now so I will update that in the documentation and say landscape mode is not fully supported on all devices.
